rest-server/README.md

```diff
@@ -463,3 +463,86 @@ Configure the rest server ip and port in [service-deployment/clusterconfig.yaml]
       "error": "JobExecuteError",
       "message": "job execute error"
     }
+
+11. `GET vcs/`
+
+    Get the list of vcs.
+
+    *Request*
+    ```
+    GET /api/v1/vcs
```

Comment:
Avoid using abbreviations unless it is really necessary: please use `GET /api/v1/virutal-clusters`.

Reply:
"error" field should be string type according to readme, but the default error handler returns a dict instead. Inconsistent json filed type will result to deserialization failure.